modules/Authentication.php

```diff
@@ -1,11 +1,17 @@
 <?php
 	class Authentication {
-		private $user;
+		public $user;
 
-		function __construct($email, $password) {
-			if (!($email && $password)) {
-				return false;
+		public function __construct($email, $password) {
+			if (!$email || !$password) {
+				return NULL;
```

Comment:
I read it somewhere, it is advised to throw an exception when it is truly an error, here what I am doing is making sure that the object doesn't exist in case the email and password are not provided.